### ISSUE TITLE
Doesn't work after restarting system. Macbook m1 pro (Ventura)

### ISSUE THREAD

**Issue by stepn1k** [norflin321/fn-lang-switcher#9]
OS doesn't run script after restarting system. 
only works if launch it manually

**Comment by agapovk** [comment]
Same problem

**Comment by tonku321** [comment]
I also have the same problem, and so far haven't been able to figure out what's going on. I have updated the plist file so that it logs errors to the fn.error file, but there are no errors after restarting the system. @Saba-Sabato do you have any idea what could be causing the issue?

**Comment by stepn1k** [comment]
Is there any news, guys?

**Comment by Saba-Sabato** [comment]
> I also have the same problem, and so far haven't been able to figure out what's going on. I have updated the plist file so that it logs errors to the fn.error file, but there are no errors after restarting the system. @Saba-Sabato do you have any idea what could be causing the issue?

Yes, in https://github.com/norflin321/fn-lang-switcher/commit/b2c655e975a42b3d8c9a3ef404111056dfe81221 you used a relative error path, but it must be absolute - cannot contain `~`. Use something like `/tmp/fn.error.log` instead. @norflin321 

As for why the original solution failed to work, I'm honestly not sure? But I just went down a rabbit-hole of `launchctl`:
https://babodee.wordpress.com/2016/04/09/launchctl-2-0-syntax/
https://gist.github.com/masklinn/a532dfe55bdeab3d60ab8e46ccc38a68
https://macblog.org/uid/
It seems like we should use some combination of `launchctl bootstrap` & `launchctl kickstart`, but it might be a while before I have time to figure it out.
